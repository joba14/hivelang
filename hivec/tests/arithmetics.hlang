
// Description:
//     Testing arithmetic operations.
// 
// Expectations:
//     The program should produce this output: "30\n10\n200\n2\n1\n"

procedure main do
	20 10 add printn
	20 10 subtract printn
	20 10 multiply printn
	20 10 divide printn
	5 2 modulus printn
end
