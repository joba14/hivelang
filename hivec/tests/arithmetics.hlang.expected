30
10
200
2
1
